src/log/Logging.cpp

```diff
@@ -38,16 +38,19 @@ logAdapter* logAdapter::getLogInstance() {
 }
 
 logAdapter::logAdapter() : m_logLevel(eLOG_LEVEL_INFO) {
+  setLogDir();
   string homeDir(UtilAll::getHomeDirectory());
-  homeDir.append("/logs/rocketmq-cpp/");
+  homeDir.append(m_log_dir);
   m_logFile += homeDir;
-  std::string fileName = UtilAll::to_string(getpid()) + "_" + "rocketmq-cpp.log.%N";
+  std::string fileName = "rocketmq_client.log";
   m_logFile += fileName;
 
   // boost::log::expressions::attr<
   // boost::log::attributes::current_thread_id::value_type>("ThreadID");
   boost::log::register_simple_formatter_factory<boost::log::trivial::severity_level, char>("Severity");
-  m_logSink = logging::add_file_log(keywords::file_name = m_logFile, keywords::rotation_size = 100 * 1024 * 1024,
+  m_logSink = logging::add_file_log(keywords::file_name = m_logFile,
+                                    keywords::target_file_name = "rocketmq_client_%Y%m%d-%N.log",
+                                    keywords::rotation_size = 10 * 1024,
```

Comment:
These are breaking changes and should be considered carefully. I suggest splitting these into a separate PR.

Reply:
Here we use boost 1.78.0 to make the backup log file name configurable. Did you mean split this change into another PR?

Reply:
Make log name compatible with RocketMQ main project, and rotation size.

Reply:
OK， I will put modification of rotation size in another PR.